bandit/core/issue.py

```diff
@@ -127,3 +129,23 @@ def issue_from_dict(data):
     i = Issue(severity=data["issue_severity"])
     i.from_dict(data)
     return i
+
+
+class HashableIssue(object):
+    def __init__(self, issue):
+        '''A hashable issue class cseline mparisions between'''
+        self.issue = issue
+
+    def __hash__(self):
+        issue_hash = 0
+        for prop in ['text', 'severity', 'confidence', 'fname', 'test',
+                     'test_id']:
+            issue_hash ^= hash(self.issue.__dict__[prop])
```

Comment:
Why not to use getattr here:
hash(getattr(self.issue(prop))
?

Reply:
agreed, updated this.